src/mqueryfront/src/components/FilterIcon.js

```diff
@@ -17,7 +17,7 @@ const FilterIcon = (props) => {
 
 FilterIcon.defaultProps = {
     size: "xs",
-    color: "black",
+    color: undefined,
```

Comment:
I think we can just delete that defaultProp. Undefined is the same as missing key.
```suggestion
```